src/Accord.Bot/Helpers/DiscordPermissionHelper.cs

```diff
@@ -0,0 +1,163 @@
+using System.Linq;
+using System.Numerics;
+using System.Threading.Tasks;
+using Remora.Discord.API.Abstractions.Objects;
+using Remora.Discord.API.Objects;
+
+namespace Accord.Bot.Helpers
+{
+    public enum DiscordPermissionType
+    {
+        Allow,
+        Deny,
+        All
+    }
```

Comment:
Probably worth making a new folder and plopping these two into separate files in that folder